Hong Kong TOPSKY.prf

```diff
@@ -62,12 +62,12 @@ Settings	sector	\Data\Sector\Hong-Kong-Sector-File.sct
 Plugins	Plugin0Display1	SMR radar display
 Settings	SettingsfilePLUGINS	\Data\Settings\GeneralSettings.txt
 Plugins	Plugin2Display1	SMR radar display
-RecentFiles	Recent1	\Data\ASR\Hong Kong ACC.asr
-RecentFiles	Recent2	\Data\ASR\Hong Kong GMC.asr
-RecentFiles	Recent3	\Data\ASR\Hong Kong TMA.asr
-RecentFiles	Recent4	\Data\ASR\Kai Tak GMC.asr
-RecentFiles	Recent5	\Data\ASR\Macau GMC.asr
-RecentFiles	Recent6	\Data\ASR\Hong Kong Zone.asr
+RecentFiles	Recent1	\Data\ASR\Kai Tak GMC.asr
+RecentFiles	Recent2	\Data\ASR\Hong Kong ACC.asr
+RecentFiles	Recent3	\Data\ASR\Hong Kong Zone.asr
+RecentFiles	Recent4	\Data\ASR\Macau GMC.asr
+RecentFiles	Recent5	\Data\ASR\Hong Kong GMC.asr
+RecentFiles	Recent6	\Data\ASR\Hong Kong TMA.asr
```

Comment:
Revert the changes to this file to avoid unnecessary conflicts when people update with the installer

Reply:
Sorry - didn't realise I committed this